### DISCUSSION TITLE
dodanie do zasobu /order/checkout-forms/{id} pola ze stawką VAT produktu

### DISCUSSION THREAD

**Discussion by PhotoSoft** [allegro/allegro-api#6958] [Suggestions - resolved]
Proszę o dodanie do zasobu /order/checkout-forms/{id} pola ze stawką VAT produktu.

**Comment by michal-kulisiewicz-radgost** [comment]
Przyłączam się do prośby.

**Comment by michal-kulisiewicz-radgost** [comment]
Proszę o informację na temat planów z dodaniem stawki do zasobu.

**Reply by PrzemyslawLukanowski** [reply]
Na tę chwilę nie mamy tego w planach, jednak jeśli się to zmieni, na pewno zamieścimy odpowiedni news na naszej stronie i forum.

**Comment by AureliuszBrussy** [comment]
Właśnie wdrożyliśmy stawkę VAT w zamówieniach: https://github.com/allegro/allegro-api/issues/8747
